src/it/java/com/google/checkstyle/test/chapter2filebasic/rule233nonascii/NonASCIICharactersTest.java

```diff
@@ -19,15 +19,12 @@
 
 package com.google.checkstyle.test.chapter2filebasic.rule233nonascii;
 
-import static com.puppycrawl.tools.checkstyle.checks.AvoidEscapedUnicodeCharactersCheck.MSG_KEY;
-
 import org.junit.jupiter.api.Test;
 
 import com.google.checkstyle.test.base.AbstractGoogleModuleTestSupport;
-import com.puppycrawl.tools.checkstyle.api.Configuration;
-import com.puppycrawl.tools.checkstyle.checks.AvoidEscapedUnicodeCharactersCheck;
 
-public class AvoidEscapedUnicodeCharactersTest extends AbstractGoogleModuleTestSupport {
+// -@cs[AbbreviationAsWordInName] Test should be named as its section name.
+public class NonASCIICharactersTest extends AbstractGoogleModuleTestSupport {
```

Comment:
can we relax validation to do lowercase before compare or to do camel case(keep first capital, all other is lowercase) for words?

Reply:
done 
https://github.com/checkstyle/checkstyle/blob/5c7e58eae3a3610dd69b5985dfc25b5c4ece8b54/src/it/java/com/google/checkstyle/test/chapter2filebasic/rule233nonascii/NonAsciiCharactersTest.java#L26

Reply:
also removed the suppression